src/reducers/profile-view.js

```diff
@@ -196,17 +221,16 @@ function viewOptionsPerThread(
     }
     case 'ADD_TRANSFORM_TO_STACK': {
       const { threadIndex, transform, transformedThread } = action;
-      const expandedCallNodePaths = uniqWith(
-        state[threadIndex].expandedCallNodePaths
+      const expandedCallNodePaths = new PathSet(
+        Array.from(state[threadIndex].expandedCallNodePaths)
```

Comment:
I'm curious, do we ever see reducers show up in performance profiles? I can't recall ever seeing one.

Reply:
I'll admit I haven't tried transforms while I was working on this patch, and actually I don't recall trying transforms while profiling so I haven't seen them ever :)

But I think you're right, what we usually see is the consequence of the reducers, especially rerenders (in the SELECT case for example). Looks like our selectors themselves are pretty cheap :)

Reply:
So, when I "expand all" before running a transform, I can see this reducer in profiles. But the process never actually finishes before this PR ;)